python/src/scipp/compat/mantid.py

```diff
@@ -111,44 +111,53 @@ def set_bin_masks(bin_masks, dim, index, masked_bins):
         bin_masks[sc.Dim.Spectrum, index][dim, masked_bin].value = True
 
 
+def validate_and_get_unit(unit):
+    known_units = {
+        "DeltaE": [sc.Dim.EnergyTransfer, sc.units.meV],
+        "TOF": [sc.Dim.Tof, sc.units.us],
+        "Wavelength": [sc.Dim.Wavelength, sc.units.angstrom],
+        "Energy": [sc.Dim.Energy, sc.units.meV],
+        "dSpacing": [sc.Dim.DSpacing, sc.units.angstrom],
+        "MomentumTransfer": [sc.Dim.Q, sc.units.dimensionless / sc.units.angstrom],
+        "QSquared": [sc.Dim.Q, sc.units.dimensionless / (sc.units.angstrom * sc.units.angstrom)],
+        "Label": [sc.Dim.Spectrum, sc.units.dimensionless],
```

Comment:
What is the reason behind using `Dim.Spectrum` here? When does Mantid use this unit?

Reply:
This is the unit that corresponds to spectrum number, I believe it may also be used for other general non-quantities but couldn't find an example to hand.